### DISCUSSION TITLE
400,403 pageui improvements

### DISCUSSION THREAD

**Discussion by dalso0418** [ZoeyVid/NPMplus#490] [Q&A] [answered]
Hello.
Crowdsec's blocking page is exposed normally, but if it is blocked by modsecurity, there is no separate blocking page.

I wish the same ui could be exposed or specified.

Thank you :)

*This content was written through a translator, so please understand even if there are mistranslations.

**Comment by Zoey2936** [comment, marked as answer]
please see here: https://nginx.org/en/docs/http/ngx_http_core_module.html#error_page
